src/pre_defined_classes/simple_programs_taking_string_user_input/hard/q10/PasswordWeakness.java

```diff
@@ -0,0 +1,43 @@
+package pre_defined_classes.simple_programs_taking_string_user_input.hard.q10;
+
+/*
+Passwords are one of the most important things that everyone in the digital world has to take care of. Some of the most
+popular passwords that are easily guessed by hackers contain the word "password" or the numbers "1234". Make a program
+that asks the user to enter a password, then checks if the password is strong or not. For this question, a password is
+weak if it can be easily guessed based on the criteria above. The program will print true or false depending on whether
```

Comment:
I would expand on this to specify this means it contains the two parameters above. This can just be done in brackets.